graylog2-web-interface/src/components/lookup-tables/adapters/index.js

```diff
@@ -21,11 +24,11 @@ PluginStore.register(new PluginManifest({}, {
       documentationComponent: CSVFileAdapterDocumentation,
     },
     {
-      type: 'httpjsonpath',
-      displayName: 'HTTP JSONPath',
-      formComponent: HTTPJSONPathAdapterFieldSet,
-      summaryComponent: HTTPJSONPathAdapterSummary,
-      documentationComponent: HTTPJSONPathAdapterDocumentation,
+      type: 'dnslookup',
```

Comment:
Note that I have changed the plugin name from 'dns-lookup' to 'dnslookup' for consistency with other adaptors. This means the @lennartkoopmann and @bernd will need to recreate the DNS Lookup adaptor for their test environments once merged.

I thought it made sense to maintain consistency since this is pre-release. I would not have renamed this if had released.

Reply:
Yes, good idea! Thanks for the heads-up.